README.md

```diff
@@ -1,42 +1,589 @@
+    * `security_group_id`: security group id.
+
+**Runtime properties:**
+* `nsx_auth`: merged copy of `nsx_auth`;
+* `resource_id`: Internal id used in plugin for work with `dynamic_set`;
+* `dynamic_member`: merged copy of `dynamic_member`.
+
+**Examples:**
+
+```
+  update_dynamic_members:
+    type: cloudify.nsx.security_group_dynamic_member
+    properties:
+      nsx_auth: <authentication credentials for nsx>
+    interfaces:
+      cloudify.interfaces.lifecycle:
+        create:
+          inputs:
+            dynamic_member:
+              security_group_id: <security group id>
+              dynamic_set:
+                - operator: OR
+                  dynamicCriteria:
+                    operator: AND
+                    isValid: true
+                    key: VM.GUEST_OS_FULL_NAME
+                    value: Server
+                    criteria: contains
+                - operator: OR
+                  dynamicCriteria:
+                    operator: AND
+                    isValid: true
+                    key: VM.GUEST_OS_FULL_NAME
+                    value: Teapot
+                    criteria: contains
+```
+* [Simple example](tests/platformtests/resources/security_groups.yaml):
+* For more complicated example look to [security_functionality.yaml](tests/integration/resources/security_functionality.yaml)
+
+### cloudify.nsx.security_group_member
+
+Attach member to [Security Group](README.md#cloudifynsxsecurity_group).
+
+**Derived From:** cloudify.nodes.ApplicationModule
+**Properties:**
+* `nsx_auth`: look above for information;
+* `group_member`:
+    * `objectId`: member id, can be other security group or vm;
+    * `security_group_id`: security group id.
+
+**Runtime properties:**
+* `nsx_auth`: merged copy of `nsx_auth`;
+* `resource_id`: Internal id used in plugin for work with `group_member`;
+* `group_member`: merged copy of `group_member`.
+
+**Examples:**
+
+```
+  slave_master_security_group_bind:
+    type: cloudify.nsx.security_group_member
+    properties:
+      nsx_auth: <authentication credentials for nsx>
+    interfaces:
+      cloudify.interfaces.lifecycle:
+        create:
+          inputs:
+            group_member:
+              security_group_id: <security group id>
+              objectId: <object Id>
+```
+
+* [Simple example](tests/platformtests/resources/security_groups.yaml):
+* For more complicated example look to [security_functionality.yaml](tests/integration/resources/security_functionality.yaml)
+
+### cloudify.nsx.security_group_exclude_member
+
+Set some object as explicitly excluded from [Security Group](README.md#cloudifynsxsecurity_group).
+
+**Derived From:** cloudify.nodes.ApplicationModule
+**Properties:**
+* `nsx_auth`: look above for information;
+* `group_exclude_member`:
+    * `objectId`: member id, can be other security group or vm;
+    * `security_group_id`: security group id.
```

Comment:
Security group ID

Though... what is this for? The previous line suggests that this can be set in the object ID? Or is this the parent security group? Needs to be covered in the docs.